streaming/streaming.go

```diff
@@ -22,9 +22,9 @@ import (
 	"time"
 
 	dataflow "cloud.google.com/go/dataflow/apiv1beta3"
-	datastream "cloud.google.com/go/datastream/apiv1alpha1"
```

Comment:
does this library change affect other streaming migrations also ?

Reply:
yes same library is used across all the streaming migrations, the older library didn't have postgres source support